src/css/utils/flex/index.css

```
@@ -1 +1,2 @@
-@import './flex';
+@import './flex-responsive.css';
+@import './flex';
```

@mgallant-aptuitiv This is really minor, but the imports should be alphabeticsl

```suggestion
@import './flex';
@import './flex-responsive.css';
```